eap74-rhel8-payg-multivm/src/main/scripts/jbosseap-setup-redhat.sh

```diff
@@ -1,8 +1,14 @@
 #!/bin/bash
 
+log() {
+    while IFS= read -r line; do
+        printf '%s %s\n' "$(date "+%Y-%m-%d %H:%M:%S")" "$line" >> /var/log/jbosseap.install.log
+    done
+}
+
```

Comment:
As discussed in review meeting, please make it so this PR does not have the debugging log file changes.